pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -1,67 +1,64 @@
 { stdenv, fetchurl, openssl, python2, zlib, libuv, v8, utillinux, http-parser
-, pkgconfig, runCommand, which, libtool
-, version
-, sha256 ? null
-, src ? fetchurl { url = "https://nodejs.org/download/release/v${version}/node-v${version}.tar.xz"; inherit sha256; }
-, preBuild ? ""
-, extraConfigFlags ? []
-, extraBuildInputs ? []
-, patches ? [],
- ...
+, pkgconfig, runCommand, which, libtool, fetchpatch
+, callPackage
+, darwin ? null
+, disableNpm ? false
 }:
 
-assert stdenv.system != "armv5tel-linux";
-
 let
 
-  deps = {
-    inherit openssl zlib libuv;
-  } // (stdenv.lib.optionalAttrs (!stdenv.isDarwin) {
-    inherit http-parser;
-  });
+  inherit (darwin.apple_sdk.frameworks) CoreServices ApplicationServices;
+  inherit (stdenv.lib) getLib optionalString optionals optionalAttrs concatMap maintainers licenses platforms;
```

Comment:
`with lib;` or `with stdenv.lib;` seems to be more common than explicitly inheriting.